tests/css/Parser.js

```diff
@@ -1528,6 +1528,18 @@
             Assert.isTrue(true);  //just don't want an error
         },
 
+        testMediaWithSimpleExpressionWithSpaces: function(){
+            var parser = new Parser({ strict: true});
+            var result = parser.parse("@media ( max-width:400px ) { }");
+            Assert.isTrue(true);  //just don't want an error
```

Comment:
Is there any way to assert something more meaningful? Could you test error count?